app/src/main/java/com/yeonae/chamelezone/view/mypage/mycourse/MyCourseActivity.kt

```diff
@@ -56,6 +53,11 @@ class MyCourseActivity : AppCompatActivity() {
         }
     }
 
+    private fun bottomSheet() {
```

Comment:
```suggestion
    private fun showBottomSheet() {
```
함수의 행동에 대한 네이밍을 명시적으로 작성하면 좋을 것 같아요.

Reply:
수정했습니당